fluff/overworld.md

```diff
@@ -32,7 +32,8 @@
 
 ### Burning Grass
 
-Burning grass has standard ICD - 3 hits/2.5 seconds.  
+Overworld grass generally takes 2 units of pyro to ignite. However, some abilities do not interact with grass.\
```

Comment:
typo, remove the "\"

Reply:
The \ is a line break

Reply:
lol my bad, i'm so used to the double spaces i forgot about the \